### PR TITLE
Fix map icons rendering

### PR DESCRIPTION
**Pull request by Siilwyn**
Mapbox does not allow loading vector images so a hack was used to load
in image data. However it seems this hack does not work in every
browser. Instead we now load image data from exported bitmap files.

https://trello.com/c/VnoZ3WIf/132-the-map-busyness-indicators-and-building-icons-are-not-visible-in-firefox

As a bonus icons now seem to load faster (on Safari & Chrome too).

### CODE REVIEW

File: src/components/FilterMenu/FilterMenu.vue [hunk 1]
```diff
@@ -180,6 +180,9 @@ function getRelativeTimeString(date: Date): string {
 
 const updatedAt = ref();
 const { resume, pause } = useIntervalFn(() => {
+  if (!spacesStore.updatedAt)
+    return;
+
```
Comment (Siilwyn) [inline src/components/FilterMenu/FilterMenu.vue:185]: Unrelated change, sorry! Small fix to prevent a race condition, I've seen this being undefined during development.

**Comment by Siilwyn** [comment]
https://trello.com/c/VnoZ3WIf/132-the-map-busyness-indicators-and-building-icons-are-not-visible-in-firefox

**Review by jbmoelker — APPROVED** [review]
I would document in the code base why the change to bitmap was made so that we don't have this issue again in the future.
